nameko/web/server.py

```diff
@@ -93,6 +93,7 @@ def start(self):
     def stop(self):
         self._is_accepting = False
         self._gt.kill()
+        self._sock.close()
```

Comment:
Actually this is fine. The ``WebServer`` is a subextension so it's stopped after all the workers have gracefully completed. By the time this method is called there won't be any in-flight calls.